packages/apps/kubernetes/values.yaml

```diff
@@ -3,11 +3,13 @@
 ## @param host The hostname used to access the Kubernetes cluster externally (defaults to using the cluster name as a subdomain for the tenant host).
 ## @param controlPlane.replicas Number of replicas for Kubernetes contorl-plane components
 ## @param storageClass StorageClass used to store user data
+## @param disabledDNSAddon Is addon KamajiControlPlane.spec.addons.coreDNS disabled
 ##
 host: ""
 controlPlane:
   replicas: 2
 storageClass: replicated
+disabledDNSAddon: false
```

Comment:
Can we disable it by default (don't provide such option for the user) and just using helm chart for coredns every time?